src/juju/jimm/api.ts

```diff
@@ -1,5 +1,11 @@
+const jimmEndpoint =
```

Comment:
might be good to add tests for this in the future, covering all possible conditions.

Reply:
Thanks for the review! I've created the task for it [here](https://warthogs.atlassian.net/browse/WD-15560) and explained the potential issue that might arise when testing it.